tests/kserve_models_web_application_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euxo pipefail
+
+
+KF_PROFILE=${1:-kubeflow-user-example-com}
+TOKEN="$(kubectl -n $KF_PROFILE create token default-editor)"
+BASE_URL="localhost:8080/kserve-endpoints"
+
+cat <<EOF | kubectl apply -f -
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-iris-private"
```

Comment:
why do we have the -private suffix? what is the justified benefit of having it ?